.builders/scripts/build_wheels.py

```diff
@@ -77,6 +79,7 @@ def main():
 
         # Off is on, see: https://github.com/pypa/pip/issues/5735
         env_vars['PIP_NO_BUILD_ISOLATION'] = '0'
+        env_vars['PIP_NO_BINARY'] = ' '.join(PACKAGES_FORCE_BUILD)
```

Comment:
Since this is builder-specific similar to `PIP_CONSTRAINT` let's add that to the images themselves as required.

Reply:
Isn't this a bit similar to what we do here? https://github.com/DataDog/integrations-core/blob/47f797831fc7754aec943c93301a1c45f9055469/.builders/scripts/repair_wheels.py#L59-L66